src/pages/docs/features/notifications.md

```diff
@@ -2,5 +2,31 @@
 title: Notifications
 description: Using the Cirro notification system.
 ---
+## What are Cirro notifications?
+Communicating with your users is essential in order to maintain a well-functioning web application, thus a crowd-working service. In addition to the Cirro provided tools to make identity management, gig management, and invitation management simpler for you and your workers, we equip you with multiple means of communication.
```

Comment:
replace term "workers" with users; you might need to be more specific at times when you refer to "users" in the context of Cirro and spaces - since spaces have different groups of users they entertain

Reply:
`users` is actually correct here I believe. As I described in the Terminology section, `users` are everyone using your Space, `workers` are only users who get invited and perform work.
You definitely can use Notifications to contact any user, not only workers.

Reply:
or did I misunderstand your comment?

Reply:
Ah, I think I understand now, you were referring to the use of `worker` at the end of the paragraph. Then I agree :D

Reply:
Rewritten section based on the comments, it's currently pending review.